project/Dependencies.scala

```diff
@@ -17,10 +17,10 @@ object Dependencies {
   val sslConfigVersion = "0.2.2"
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
-  val aeronVersion = "1.3.0"
+  val aeronVersion = "1.5.1"
```

Comment:
I think we should wait with this update. It will probably work, but we can stay a bit behind bleeding edge. There is a 2.4.1 that we could move to first.